### DISCUSSION TITLE
Publishing git hub pages

### DISCUSSION THREAD

**Discussion by mohamedbassim** [community/community#22544] [Other Feature Feedback, Questions, & Ideas]
<p>i have a site where i have a source code for server-side code in the repo and when i publish the site through github pages it actually shows only the client side code and not the server side code</p>
<p>Note: the server-side code file is actually out of the folder of the natural css,html and javascript client-side code files because these are in a lonely folder called website and the server-side code is out of them so how can i get github pages to show read that server.js file that i have ?</p>

**Comment by airtower-luna** [comment]
<aside class="quote no-group" data-username="mohamedbassim" data-post="1" data-topic="227522">
<div class="title">
<div class="quote-controls"></div>
 mohamedbassim:</div>
<blockquote>
<p>the server-side code is out of them so how can i get github pages to show read that server.js file that i have ?</p>
</blockquote>
</aside>
<p>If by “read” you mean run the server application, the answer is unfortunately “you can’t”. GitHub Pages is for static sites only, if you want to run a server side application you’ll have to find another host.</p>

**Comment by mohamedbassim** [comment]
<p>thx for the information ididnt know that<br>
but can you recommend another host which would be good as a start for me</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>That depends on what you need from the host. How do you want to run your application, what kind of base do you need from the host? If you describe those things, maybe someone here will have an idea.</p>

**Comment by mohamedbassim** [comment]
<p>ok thx for help on this question :slight_smile:</p>
